src/lib/expansion/expansion-panel.ts

```diff
@@ -109,17 +111,12 @@ export class MatExpansionPanel extends CdkAccordionItem
     this.accordion = accordion;
   }
 
-  /** Whether the expansion indicator should be hidden. */
-  _getHideToggle(): boolean {
-    if (this.accordion) {
-      return this.accordion.hideToggle;
-    }
-    return this.hideToggle;
-  }
-
   /** Determines whether the expansion panel should have spacing between it and its siblings. */
   _hasSpacing(): boolean {
     if (this.accordion) {
+      // We don't need to subscribe to the `stateChanges` of the parent accordion because each time
```

Comment:
This seems like a weird place to put this comment, but I don't have any better suggestions.